### PR TITLE
blog post about kiali deprecation

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**

Changes proposed in this pull request:

- Preparing the kiali deprecation with a public blog post so that it can be referenced in the release notes
- ...
- ...

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/15412

### CODE REVIEW

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+tags:
+  - kyma
+  - observability
+  - Kiali
+
+redirectFrom:
+  - "/blog/kiali-deprecation"
+---
+
+I’m part of the Kyma team working on the Observability feature, and I like to inform you today that we decided to deprecate the Kyma Kiali component and like t to provide you some reasoning behind that decision.
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:14]:
```suggestion
As part of the Kyma team working on the Observability feature, I'd like to let you know that we decided to deprecate the Kyma Kiali component. In this blog post, I'd like to provide you our reasoning behind that decision.
```

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Motivation
+
+In the recent [blog post](https://kyma-project.io/blog/2022/9/21/observability-strategy/) I outlined the new focus for the Observability domain in Kyma. To summarize, the shift into providing enterprise-grade qualities for Kyma modules moved the focus to enable the user streaming telemetry data into existing observability backends. Hereby, Kyma explicitly will not focus on being yet another way of running a specific observability stacks in enterprise-grade qualities but focusing on an open and easy integration into existing solutions.
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:18]:
```suggestion
In my recent [blog post](https://kyma-project.io/blog/2022/9/21/observability-strategy/), I outlined the new strategy for the Observability domain in Kyma. To summarize, the shift towards providing enterprise-grade qualities for Kyma modules moved the focus to enable the users to stream telemetry data into their existing observability backends. Hereby, Kyma explicitly decided to support open and easy integration into existing solutions instead of being yet another provider of a specific observability stack in enterprise-grade qualities.
```
Just guessing :D 
- I tried to get rid of the repeated use of "focus" but during rephrasing, I may have misrepresented which parts you want to stress.
- I wanted to use the word "strategy" so readers easily recognize which blog post you mean, even without clicking the link

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+The same principle we will apply now to Kiali. The current Kiali component is very lightweight and does not provide well-defined configuration options at runtime. The Kyma team supports the Kiali project, but don't want to become an active contributor with the goal of running Kiali with managed qualities inside a Kyma cluster. Instead, Kyma needs to assure that the user is at anytime enabled to integrate with Kiali, if they want to.
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:20]:
```suggestion
Now, we'll apply the same principle to Kiali. Kyma's current Kiali component is very lightweight and does not provide well-defined configuration options at runtime. The Kyma team supports the Kiali project, but doesn't want to become an active contributor with the goal of running Kiali with managed qualities inside a Kyma cluster. Instead, Kyma wants to assure that the users can connect with Kiali whenever they want.
```

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## The Facts & Alternatives
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:22]:
```suggestion
## The Facts and Alternatives
```

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+The Kiali component will be marked as deprecated with Kyma version 2.8. Instructions on how to install Kiali on your own are already [provided](https://github.com/kyma-project/examples/tree/main/kiali) and maintained on a best effort base. After you transitioned to a custom installation, the Kyma component gets removed with Kyma 2.11.
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:24]:
```suggestion
The Kiali component will be marked as deprecated with Kyma version 2.8. 
Instructions on [how to install Kiali on your own](https://github.com/kyma-project/examples/tree/main/kiali) have already been provided and will be maintained on a best-effort base. After you transitioned to a custom Kiali installation, Kyma's Kiali component is planned to be removed with Kyma 2.11.
```

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+### Conclusion
+
+To sharpen the focus of the project the Kiali component gets transformed into a simple set of instructions. You can start the transformation already today.
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:28]:
```suggestion
To sharpen the focus of the Kyma project, the Kiali component is transformed into a simple set of instructions. You can start the transformation already today.
```

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md (hunk 4 above)
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:22]:
```suggestion
## The Future
```

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+The same principle we will apply now to Kiali. The current Kiali component is very lightweight and does not provide well-defined configuration options at runtime. The Kyma team supports the Kiali project, but don't want to become an active contributor with the goal of running Kiali with managed qualities inside a Kyma cluster. Instead, Kyma needs to assure that the user is at anytime enabled to integrate with Kiali, if they want to.
+
+## The Facts & Alternatives
+
+The Kiali component will be marked as deprecated with Kyma version 2.8. Instructions on how to install Kiali on your own are already [provided](https://github.com/kyma-project/examples/tree/main/kiali) and maintained on a best effort base. After you transitioned to a custom installation, the Kyma component gets removed with Kyma 2.11.
+
+### Conclusion
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:26]:
```suggestion
```

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+  - observability
+  - Kiali
+
+redirectFrom:
+  - "/blog/kiali-deprecation"
+---
+
+I’m part of the Kyma team working on the Observability feature, and I like to inform you today that we decided to deprecate the Kyma Kiali component and like t to provide you some reasoning behind that decision.
+
+## Motivation
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:16]:
```suggestion
## The Background
```

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+tags:
+  - kyma
+  - observability
+  - Kiali
+
+redirectFrom:
+  - "/blog/kiali-deprecation"
+---
+
+As part of the Kyma team working on the Observability feature, I'd like to let you know that we decided to deprecate the Kyma Kiali component. In this blog post, I'd like to provide you our reasoning behind that decision.
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:14]:
```suggestion
As part of the Kyma team working on the Observability feature, I'd like to let you know that we decided to deprecate the Kyma Kiali component. In this blog post, I'm going to give you our reasoning behind that decision.
```
didn't notice the Wortwiederholung before

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+## The Background
+
+In my recent [blog post](https://kyma-project.io/blog/2022/9/21/observability-strategy/), I outlined the new strategy for the Observability domain in Kyma. To summarize, the shift towards providing enterprise-grade qualities for Kyma modules moved the focus to enable the users to stream telemetry data into their existing observability backends. Hereby, Kyma explicitly decided to support open and easy integration into existing solutions instead of being yet another provider of a specific observability stack in enterprise-grade qualities.
```
Comment (NHingerl) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:18]:
```suggestion
In my recent [blog post](https://kyma-project.io/blog/2022/9/21/observability-strategy/), I outlined the new strategy for the Observability domain in Kyma. To summarize, the shift towards providing enterprise-grade qualities for Kyma modules moved the focus to enable the users to stream telemetry data into their existing observability backends. Hereby, Kyma explicitly decided to support open and easy integration into existing solutions instead of being yet another provider of a specific observability stack in enterprise-grade quality.
```

**Review by NHingerl — DISMISSED** [review]

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+tags:
+  - kyma
+  - observability
+  - Kiali
+
+redirectFrom:
+  - "/blog/kiali-deprecation"
+---
+
+As part of the Kyma team working on the Observability feature, I'd like to let you know that we decided to deprecate the Kyma Kiali component. In this blog post, I'm going to give you our reasoning behind that decision.
```
Comment (varbanv) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:14]: I would change "Observability feature" to "Observability capabilites" since it's not really a single feature.

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+## The Background
+
+In my recent [blog post](https://kyma-project.io/blog/2022/9/21/observability-strategy/), I outlined the new strategy for the Observability domain in Kyma. To summarize, the shift towards providing enterprise-grade qualities for Kyma modules moved the focus to enable the users to stream telemetry data into their existing observability backends. Hereby, Kyma explicitly decided to support open and easy integration into existing solutions instead of being yet another provider of a specific observability stack in enterprise-grade quality.
```
Comment (varbanv) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:18]: I would rephrase the summary more like:
"To summarize: the shift towards providing enterprise-grade qualities for Kyma modules changed the focus to enabling users to stream telemetry data into centralized enterprise-grade observability backends.

File: content/blog-posts/2022-09-30-Kiali-deprecation/index.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+
+Now, we'll apply the same principle to Kiali. Kyma's current Kiali component is very lightweight and does not provide well-defined configuration options at runtime. The Kyma team supports the Kiali project, but doesn't want to become an active contributor with the goal of running Kiali with managed qualities inside a Kyma cluster. Instead, Kyma wants to assure that the users can connect with Kiali whenever they want.
```
Comment (varbanv) [inline content/blog-posts/2022-09-30-Kiali-deprecation/index.md:20]: I would rephrase this a bit to put the focus on how running an in-cluster component doesn't fit the strategy:
"Taking into consideration that, currently, the Kiali component is not enterprise-grade quality and is not centralized, we concluded that the current approach does not fit our strategy. Thus, we want to enable users to easily connect all their Kyma clusters to a central Kiali (or other) instance rather than maintain individual instances in each cluster."

Comment (a-thaler) [reply content/blog-posts/2022-09-30-Kiali-deprecation/index.md]: The argumentation with recommending a managed offering does not really apply to kiali. I cannot find something like a managed offering (as it requires the metrics in a prometheus and access to the apiserver). So usually you would deploy it along with istio inside the cluster. So I would keep the same argumentation that it is simply not our focus to provide such service in high quality.

File: content/blog-posts/2022-10-10-Kiali-deprecation/index.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+
+## The Background
+
+In my recent [blog post](https://kyma-project.io/blog/2022/9/21/observability-strategy/), I outlined the new strategy for the Observability domain in Kyma. To summarize, the shift towards providing enterprise-grade qualities for Kyma modules moved the focus to enable the users to stream telemetry data into their centralized (outer-cluster) observability backends. Hereby, Kyma explicitly decided to support open and easy integration into existing solutions instead of being yet another provider of a specific observability stack in enterprise-grade quality.
+
+Now, we'll apply the same principle to Kiali. Kyma's current Kiali component is very lightweight and does not meet enterprise-grade qualities and with that the current approach does not fit our strategy. The Kyma team supports the Kiali project, but doesn't want to become an active member having the goal of running Kiali with managed qualities inside a Kyma cluster. Instead, Kyma wants to assure that the users can connect with Kiali whenever they want.
```
Comment (NHingerl) [inline content/blog-posts/2022-10-10-Kiali-deprecation/index.md:20]:
```suggestion
Now, we'll apply the same principle to Kiali. Kyma's current Kiali component is very lightweight and does not meet enterprise-grade qualities. Thus, the current approach does not fit our strategy. 

Instead of running Kiali with managed qualities inside a Kyma cluster, Kyma wants to enable users to connect with Kiali whenever they want. Consequently, the Kyma team supports the Kiali project but won't become an active member. 
```

Do we really need the sentence about the Kiali project and not becoming an active member?

File: content/blog-posts/2022-10-10-Kiali-deprecation/index.md [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+
+## The Future
+
+The Kiali component will be marked as deprecated with Kyma version 2.8. 
+Instructions on [how to install Kiali on your own](https://github.com/kyma-project/examples/tree/main/kiali) have already been provided and will be maintained on a best-effort base. After you transitioned to a custom Kiali installation, Kyma's Kiali component is planned to be removed with Kyma 2.11.
```
Comment (NHingerl) [inline content/blog-posts/2022-10-10-Kiali-deprecation/index.md:25]: What if not all users transition to a custom Kiali installation before 2.11? Do we postpone the removal? "After..." sounds like it is a prerequisite that must be fulfilled before we can remove it.

Comment (NHingerl) [reply content/blog-posts/2022-10-10-Kiali-deprecation/index.md:27]:
```suggestion
Instructions on [how to install Kiali on your own](https://github.com/kyma-project/examples/tree/main/kiali) have already been provided and will be maintained on a best-effort base. Kyma's Kiali component is planned to be removed with Kyma release 2.11, so you'll want to transition to a custom Kiali installation before that.
```

**Review by NHingerl — APPROVED** [review]

**Review by chrkl — APPROVED** [review]
